### PR TITLE
changelog update for 24.1

### PR DESCRIPTION
**Pull request by devoncarew**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,9 @@
+## 24.1
+- update Flutter icons
+- fix an exception when the selection changes and the outline view isn't visible
+- fix for an issue with reload on save in profile runs
+- fix for a 2017.3 issue with a 'no running apps' messing in the inspector
```
Comment (pq) [inline CHANGELOG.md]: messing => message

File: resources/META-INF/plugin.xml [hunk 2]
```diff
@@ -27,6 +27,7 @@
 
   <change-notes>
     <![CDATA[
+<h2>24.1</h2><ul><li>update Flutter icons</li><li>fix an exception when the selection changes and the outline view isn't visible</li><li>fix for an issue with reload on save in profile runs</li><li>fix for a 2017.3 issue with a 'no running apps' messing in the inspector</li></ul>
 <h2>24.0</h2><ul><li>
```
Comment (pq) [inline resources/META-INF/plugin.xml]: And here too naturally.

File: CHANGELOG.md (hunk 1 above)
Comment (devoncarew) [reply CHANGELOG.md]: oops! thanks

Comment (devoncarew) [reply CHANGELOG.md]: https://github.com/flutter/flutter-intellij/pull/2204

**Pull request merged**